### DISCUSSION TITLE
someone stole your work

### DISCUSSION THREAD

**Discussion by noobiessss** [Nain57/Smart-AutoClicker#434] [General]
https://play.google.com/store/apps/details?id=com.crtyt.autoclicker

**Comment by Nain57** [comment]
Thank you, this is really helpfull.
Can anybody seeing this can also report the application or add a bad review ? The more there is, the easier it will be to take it down.

**Comment by noobiessss** [comment]
https://support.google.com/googleplay/android-developer/contact/policy_violation_report?sjid=4303335155420151117-NC

**Reply by Nain57** [reply]
Seen directly with the account owner, it should no longer be available as seen in its app listing: https://play.google.com/store/apps/dev?id=8972423538265626279

Thanks again !

**Comment by Cyber5imon** [comment]
The offending app has been removed.
